src/gfx_layout.cpp

```diff
@@ -25,6 +25,10 @@
 #include "os/windows/string_uniscribe.h"
 #endif /* WITH_UNISCRIBE */
 
+#ifdef OSX
```

Comment:
missing `WITH_COCOA` check? Maybe a dedicated macro would be more maintainable

Reply:
I've switched all occurrences to `WITH_COCOA` only. We never defined it without also defining `OSX`, which makes the double test pointless.